app/src/main/java/com/vultisig/wallet/ui/components/buttons/VsHoldableButton.kt

```diff
@@ -64,7 +64,7 @@ fun VsHoldableButton(
                         val up = waitForUpOrCancellation()
 
                         if (up != null) {
-                            if (progress.value < 1f) onClick()
+                            if (progress.value < 0.5f) onClick()
```

Comment:
I am also thinking to actually get a reference of "scope.launch" and call `cancelAndJoin`. thoughts?